zip-0312.rst

```diff
@@ -1,7 +1,447 @@
 ::
 
   ZIP: 312
-  Title: Shielded Multisignatures using FROST
-  Status: Reserved
-  Category: Standards / RPC / Wallet
+  Title: FROST for Spend Authorization Signatures
```

Comment:
```suggestion
  Title: FROST for Spend Authorization Multisignatures
```
"FROST for Spend Authorization Signatures" doesn't tell you what it's for unless you already know what FROST is for.